my_project/global_scheme.py

```diff
@@ -133,20 +132,41 @@
     "DBT": {
         "name": "Dry bulb temperature",
         "color": ["#00b3ff", "#000082", "#ff0000", "#ffff00"],
-        "unit": "°C",
-        "range": [-40, 50],
+        "si": {
+            "unit": "°C",
+            "range": [-40, 50],
+        },
+        "ip": {
+            "unit": "°F",
+            "range": [-40, 122],
+        },
+        "conversion_function": "temperature",
```

Comment:
This is very good since now you can select the unit to display and the range as a function of the user's selection